var/spack/repos/builtin/packages/mysql/package.py

```diff
@@ -0,0 +1,95 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+import os
+
+class Mysql(Package):
+    """The MySQL software delivers a very fast, multi-threaded, multi-user,
+       and robust SQL (Structured Query Language) database server."""
+
+    homepage = "http://dev.mysql.com/"
+    url      = """http://pkgs.fedoraproject.org/repo/pkgs/community-mysql/
+                  mysql-5.7.11.tar.gz/f84d945a40ed876d10f8d5a7f4ccba32/
+                  mysql-5.7.11.tar.gz"""
```

Comment:
It does work the same for the `version` lines. For more details, see [here](https://github.com/LLNL/spack/blob/develop/lib/spack/spack/cmd/flake8.py). Also check out the [Contribution Guide](http://spack.readthedocs.io/en/latest/contribution_guide.html#flake8-tests) for instructions on running the flake8 tests locally before you even submit a PR.

For the version lines, I generally do something like:
```python
version('1.2.3', '1234567890abcdefghijklmnopqrstuvwxyz',
        url='https://super.long.url/all/on/one/line')
```